README_ro.md

```diff
@@ -0,0 +1,1188 @@
+<h1 align="center">Computer Science (Informatică)</h1>
+<h4 align="center">
+    <p>
+        <b>English</b> |
+        <a href="https://github.com/shhossain/computer_science/blob/main/README_es.md">Español</a> |
+        <a href="https://github.com/shhossain/computer_science/blob/main/README_fr.md">Français</a> |
+        <a href="https://github.com/shhossain/computer_science/blob/main/README_bn.md">বাংলা</a> |
+        <a href="https://github.com/shhossain/computer_science/blob/main/README_ta.md">தமிழ்</a> |
+        <a href="https://github.com/shhossain/computer_science/blob/main/README_guj.md">ગુજરાતી</a> |
+        <a href="https://github.com/shhossain/computer_science/blob/main/README_pt.md">Portuguese</a> |
+        <a href="https://github.com/shhossain/computer_science/blob/main/README_ro.md">Română</a>
+    </p>
+</h4>
+
+## Ghidul de contribuție
+Dacă sunteți interesat să contribuiți la acest proiect, vă rugăm să acordați un moment pentru a revizui [CONTRIBUTING.md](https://github.com/shhossain/computer_science/blob/main/CONTRIBUTING.md) pentru instrucțiuni detaliate despre cum să începeți. Contribuțiile dumneavoastră sunt foarte apreciate!
+
+<!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
+[![Toți colaboratorii](https://img.shields.io/badge/all_contributors-132-orange.svg?style=flat-square)](#contributors-)
+<!-- ALL-CONTRIBUTORS-BADGE:END -->
+
+## Cuprins
+- [Introducere](#introduction)
```

Comment:
change table of contents links:
[Introducere](#introduction) to [Introducere](#Introducere)

Reply:
Thanks for looking over my PR! I'll fix this issue everywhere it occurs.

Reply:
@shhossain I've fixed the issue and some other issues as well. I think it should be good now!